apps/civil/base/kustomization.yaml

```diff
@@ -5,5 +5,6 @@ resources:
   - ../identity/identity.yaml
   - ../civil-service/civil-service.yaml
   - ../civil-general-applications/civil-general-applications.yaml
+  - civil-values.enc.yaml
```

Comment:
Name of the secret needs to be civil-work-allocation-secret
Sorry I should have told you this yesterday